packages/indent-list/src/injectIndentListComponent.tsx

```diff
@@ -80,31 +80,27 @@ export const injectIndentListComponent = (
 
       return (
         <div className={`${className}`} style={style}>
-          {markerComponent ? (
-            markerComponent({
-              checked: checked,
-              onChange: (v: boolean) => {
-                const path = findNodePath(editor, element);
-                setNodes(editor, { checked: v }, { at: path });
-              },
-            })
-          ) : (
-            <input
-              contentEditable={false}
-              data-slate-void
-              type="checkbox"
-              style={{
-                marginRight: 5,
-                marginLeft: -17,
-                paddingTop: -10,
-              }}
-              checked={checked}
-              onChange={(v) => {
-                const path = findNodePath(editor, element);
-                setNodes(editor, { checked: v.target.checked }, { at: path });
-              }}
-            />
-          )}
+          <div contentEditable={false} data-slate-void>
```

Comment:
Adding `data-slate-void` tells Slate that the DIV represents a void element, which is untrue here. The DIV itself is no Slate element, and the enclosing Slate element is not a void. Does removing the `data-slate-void` change the behaviour at all?

Reply:
https://github.com/ianstormtaylor/slate/issues/5435 .
I remember that I will occur the above error in my applacation. Add this to fix it find in the comment.but now I remove it in Plate,There doesn't seem to be a problem